internal/table/console.go

```diff
@@ -31,7 +31,7 @@ func NewTableWriter(out io.Writer) *TabularWriter {
 func (t *TabularWriter) Write(cells ...interface{}) error {
 	t.initSize(cells)
 	// colWidth = (totalWidth - numOfSeparators) / numOfColumns
-	colWidth := (t.columnCount - (len(cells)+1)*2) / len(cells)
+	colWidth := (t.columnCount - len(cells) - 2) / len(cells)
```

Comment:
Changes in this file doesn't seem to be related to the persistence stuff. Are they leftovers?  Same comment for the `console_test.go` file.

Reply:
Yes I noticed that there were some problems with those. It would be great if we can review them 🙏